docs/guides/moodleapp/remote-themes.md

```diff
@@ -0,0 +1,401 @@
+    --item-divider-color: blue;
+
+    /* Empty Divider */
+    --spacer-background: yellow;
+}
+```
+
+### Progress bar
+
+```css
+core-progress-bar {
+    --core-progressbar-height: 4px;
+    --core-progressbar-color: red;
+    --core-progressbar-text-color: green;
+    --core-progressbar-background: blue;
+}
+```
+
+### More page
+
+```css
+/* Icons */
+page-core-mainmenu-more {
+    --core-more-icon: red;
+}
+
+/* Target a specific icon */
+page-core-mainmenu-more .addon-privatefiles-handler ion-icon {
+    color: green !important;
+}
+
+/* Items */
+page-core-mainmenu-more {
+    --core-more-item-border: blue;
+    --spacer-background: blue;
+}
+```
+
+### Login page
+
+You can personalise some colors in the Login page, but keep in mind that this only includes the credentials page (the one after you select the site).
+
+```css
+body {
+    --core-login-background: red;
+    --core-login-text-color: blue;
+    --core-login-input-background: green;
+    --core-login-input-color: yellow;
+}
+```
+
+### Messages page
+
+```css
+body {
+    --addon-messages-message-bg: white;
+    --addon-messages-message-activated-bg: gray-light;
+    --addon-messages-message-note-text: gray-dark;
+    --addon-messages-message-mine-bg: gray-light;
+    --addon-messages-message-mine-activated-bg: gray;
+    --addon-messages-discussion-badge: orange;
+    --addon-messages-discussion-badge-text: white;
+    --core-send-message-input-background: gray-light;
+    --core-send-message-input-color: black;
+}
+```
+
+### Showing course summary image on course page
+
+By default, course summary images are hidden to reduce scrolling when entering a course. If you want to change this behaviour, you can include the following CSS in your Remote Theme:
+
+```css
+ion-app core-course-format .core-format-progress-list .core-course-thumb {
+    display: block !important;
+}
+```
+
+## Updating your theme after release
+
+Once you have configured your theme, some users may already have downloaded previous styles and they will be cached.
+
+If you are updating the styles and you want users to get the latest version, you can change the url of the theme file. This doesn't mean that you need to change the name of the file itself, you can just add some query parameters that will be irrelevant when the file is downloaded:
+
+```text
+https://mysite.com/mobile/mobiletheme.css?version=1
+```
+
+Every time you make some changes in your theme and you want the file to be re-downloaded in the app, just increase this number.
+
+## Difference between Remote themes and Branded Apps
+
+Remote Theme styles can be tricky to modify. There are lots of CSS rules and some of them can change between versions. Using your own Branded App, you will have better integrations because you can also use Sass variables to change colors and styles. Additionally, you will get your custom application icon and the theming will cover the entire application, not just pages using your site.
```

Comment:
```suggestion
Remote theme styles can be tricky to modify. There are lots of CSS rules and some of them can change between versions. Using your own Branded App, you will have better integrations because you can also use Sass variables to change colors and styles. Additionally, you will get your custom application icon and the theming will cover the entire application, not just pages using your site.
```